### PR TITLE
rename target to select

### PR DESCRIPTION
**Pull request by ItsSammyM**

### CODE REVIEW

File: server/src/game/available_buttons.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ impl AvailableButtons{
                 actor_ref.alive(game) && target_ref.alive(game),
 
             target: 
-                actor_ref.can_night_target(game, target_ref) && 
+                actor_ref.can_select(game, target_ref) && 
                 game.current_phase().is_night(),
 
             day_target:
```
Comment (Jack-Papel) [inline server/src/game/available_buttons.rs:27]: `special_select` or `special`?

File: server/src/game/player/mod.rs [hunk 2]
```diff
@@ -73,7 +73,7 @@ struct PlayerNightVariables{
 
     silenced: bool,
 
-    chosen_targets: Vec<PlayerReference>,
+    selection: Vec<PlayerReference>,
```
Comment (Jack-Papel) [inline server/src/game/player/mod.rs:76]: selections?

File: server/src/game/role/mortician.rs [hunk 3]
```diff
@@ -66,7 +66,7 @@ impl RoleStateImpl for Mortician {
     fn can_day_target(self, _game: &Game, _actor_ref: PlayerReference, _target_ref: PlayerReference) -> bool {
         false
     }
-    fn convert_targets_to_visits(self, game: &Game, actor_ref: PlayerReference, target_refs: Vec<PlayerReference>) -> Vec<Visit> {
+    fn convert_selection_to_visits(self, game: &Game, actor_ref: PlayerReference, target_refs: Vec<PlayerReference>) -> Vec<Visit> {
         crate::game::role::common_role::convert_targets_to_visits(game, actor_ref, target_refs, false)
```
Comment (Jack-Papel) [inline server/src/game/role/mortician.rs:70]: selections

File: server/src/game/role/snoop.rs [hunk 4]
```diff
@@ -41,13 +41,13 @@ impl RoleStateImpl for Snoop {
         }
     }
     fn do_day_action(self, _game: &mut Game, _actor_ref: PlayerReference, _target_ref: PlayerReference) {}
-    fn can_night_target(self, game: &Game, actor_ref: PlayerReference, target_ref: PlayerReference) -> bool {
+    fn can_select(self, game: &Game, actor_ref: PlayerReference, target_ref: PlayerReference) -> bool {
         crate::game::role::common_role::can_night_target(game, actor_ref, target_ref)
```
Comment (Jack-Papel) [inline server/src/game/role/snoop.rs:45]: Look at all the common_role functions for me

File: server/tests/kit/player.rs [hunk 5]
```diff
@@ -34,14 +34,14 @@ impl TestPlayer {
     }
 
     pub fn set_night_targets(&self, targets: Vec<TestPlayer>)->bool {
```
Comment (Jack-Papel) [inline server/tests/kit/player.rs:36]: Rename

File: server/tests/kit/player.rs [hunk 6]
```diff
@@ -34,14 +34,14 @@ impl TestPlayer {
...
-        self.0.set_chosen_targets(
+        self.0.set_selection(
             game!(self), 
             targets.into_iter().map(|t|t.0).collect()
         )
     }
 
     pub fn set_night_target(&self, target: TestPlayer)->bool {
```
Comment (Jack-Papel) [inline server/tests/kit/player.rs:43]: Rename

**Review by Jack-Papel — APPROVED** [review]
If you end up changing day targets too, re-request review. Otherwise address comments.

File: server/src/game/available_buttons.rs (hunk 1 above)
Comment (ItsSammyM) [reply server/src/game/available_buttons.rs:27]: it is not selecting, thats a different PR them. Issue already exists too

File: server/src/game/player/mod.rs (hunk 2 above)
Comment (ItsSammyM) [reply server/src/game/player/mod.rs:76]: selection and selections mean almost the exact same thing

File: server/src/game/role/mortician.rs (hunk 3 above)
Comment (ItsSammyM) [reply server/src/game/role/mortician.rs:70]: i like selection better

**Pull request merged**